frontend/src/components/sidebar/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import sidebarStyles from './sidebar.module.scss';
+
+const Sidebar = () => {
+  const buttonList = [
+    { name: 'Summary' },
+    { name: 'Rota' },
+    { name: 'Employees' },
+  ];
+  return (
+    <div className={sidebarStyles.main}>
```

Comment:
If you rewrite the component the same way I've suggested in the `<Home />` component, do you need to use `sidebarStyles.main` or can you give this a className using snake case such as `sidebar-container`?

Reply:
I can do that if you like, all what im doing does is it allows the classnames in the scss file to be like` .container`  or `.button` ect and the fact that it is a module automatically handles adding the sidebar part. Its what I used in my grad project and an accepted concept

Reply:
yeah if it worked in grad project it's fine to go ahead